lib/drivers/oneTouchMiniDriver.js

```diff
@@ -486,6 +486,9 @@ module.exports = function (config) {
 
     connect: function (progress, data, cb) {
       debug('in connect!');
+
+      data.deviceInfo.deviceId = data.id;
```

Comment:
why is this change part of a ContourNext pull request? This should probably be a completely separate PR.